ig/input/fsh/ImmunizationRecommendation.fsh

```diff
@@ -2,17 +2,65 @@ Profile: VPImmunizationRecommendation
 Parent: ImmunizationRecommendation
 Id: vp-immunization-recommendation
 Title: "ImmunizationRecommendation"
-Description: "ImmunizationRecommendation profile for use in FHIR Vaccination Pass"
-* recommendation.vaccineCode 1..1 MS
-* recommendation.vaccineCode only CodeableConcept
-* recommendation.vaccineCode from http://fhir.de/CodeSystem/ifa/pzn
-* recommendation.targetDisease 1..1 MS
-* recommendation.targetDisease from http://hl7.org/fhir/sid/icd-10
-* recommendation.dateCriterion 1..1 MS
-* recommendation.extension contains VPFulfillingImmunization named fulfillingImmunization 0..1 MS
+Description: "ImmunizationRecommendation profile for use in FHIR Vaccination Pass."
+* id 0..1 MS
+* patient 1..1 MS
+* patient only Reference(VPPatient)
+* date 1..1 MS
+* recommendation 1..1 MS
+  * vaccineCode 1..1 MS
+  * vaccineCode from http://fhir.de/CodeSystem/ifa/pzn
+    * coding 1.. MS
+      * system 1..1 MS
+      * code 1..1 MS
+  * targetDisease 1..1 MS
+  * targetDisease from http://hl7.org/fhir/sid/icd-10
+    * coding 1.. MS
+      * system 1..1 MS
+      * code 1..1 MS
+  * forecastStatus 1..1 MS
+  * forecastStatus from http://hl7.org/fhir/ValueSet/immunization-recommendation-status
+    * coding 1.. MS
+      * system 1..1 MS
+      * code 1..1 MS
+  * dateCriterion 1.. MS
+    * code 1..1 MS
+    * code from http://hl7.org/fhir/ValueSet/immunization-recommendation-date-criterion
+      * coding 1.. MS
+        * system 1..1 MS
+        * code 1..1 MS
+    * value 1..1 MS
+  * supportingImmunization 0.. MS
+  * supportingImmunization only Reference(VPImmunization)
+* extension contains VPImmunizationRecommendationIsDeactivated named isDeactivated 1..1 MS
```

Comment:
This should probably be a `modifierExtension`:
- https://build.fhir.org/extensibility.html#modifierExtension
- http://hl7.org/fhir/uv/shorthand/N1/reference.html#extension-paths

Reply:
WONTFIX due to issues with tooling when `modifierExtension` is used.